iron-ajax.html

```diff
@@ -118,7 +118,7 @@
        */
       contentType: {
         type: String,
-        value: 'application/x-www-form-urlencoded'
+        value: null
```

Comment:
There is a (potentially undesired) side-effect of making this `null` by default: any observer that depends on this value will be called during initialization of the element. For example, `_requestOptionsChanged` will be called automatically when an `iron-ajax` instance is created, because `null` is assigned to `this.contentType`. See: https://github.com/PolymerElements/iron-ajax/blob/content-type/iron-ajax.html#L267-L270

In the case of `iron-ajax`, I don't think this will cause a bug immediately, but we have generally avoided default `null` unless it's really necessary.

Does the value have to be `null` by default, or will it suffice for it to default to `undefined`?